build.gradle

```diff
@@ -69,7 +70,70 @@ apply plugin: 'opensearch.opensearchplugin'
 apply plugin: 'opensearch.pluginzip'
 apply plugin: 'opensearch.rest-test'
 apply plugin: 'opensearch.testclusters'
-apply from: 'gradle/formatting.gradle'
+// apply from: 'gradle/formatting.gradle'
+
+spotless {
+    java {
+        // Normally this isn't necessary, but we have Java sources in
+        // non-standard places
+        target '**/com/amazon/dlic/**/*.java'
```

Comment:
Hm .. same as https://github.com/opensearch-project/OpenSearch/pull/7372 ?

Reply:
Hi @reta, sorry I am not sure I know what you are referencing here :/. I assume you are looking at the list of targets which I know is not the prettiest way of doing things. Unfortunately, I did not see a ton of options for creating a clearer description of which directories have the new formatting in place. 

Do you have a suggestion you would like me to follow?

Reply:
My sincere apologies, pasted wrong link here (not sure how), I meant this https://github.com/opensearch-project/security/pull/2823 - seems like doing the same changes ....

Reply:
Oh gotcha. Yeah, for whatever reason, there does not seem to be many changes within the target set here. I will rebase and expand the range to include more. It is hard to tell before rebasing what is actually new. Good catch.

Reply:
Now should have changes under `security/*` and `test/security/*`.

Looks like there were a lot of line break swaps in the tests...